src/devices/Pn532/Pn532.cs

```diff
@@ -18,6 +18,8 @@
 using Iot.Device.Rfid;
 using Iot.Device.Pn532;
 using Microsoft.Extensions.Logging;
+using SixLabors.ImageSharp.PixelFormats;
```

Comment:
This using statement seems to be in error.

Reply:
```suggestion
```